Changes

```diff
@@ -502,6 +502,10 @@ Working version
 
 ### Runtime system:
 
+- MPR#3771, GPR#153, GPR#1200: Unicode support for the Windows runtime.
+  (ygrek, Clement Franchini, Nicolas Ojeda Bar, review by Alain Frisch, David
+  Allsopp, ...)
```

Comment:
Are there more names to go here? :smile:

Reply:
More importantly, please note that this is a breaking Change for (at least) two reasons:

1. You need to configure OCaml differently to get the old, broken behaviour for Latin-1
2. The change in C API with charnat

All changes are breaking on Windows only (which might be worth noting).